README.md

```diff
@@ -0,0 +1,72 @@
+
+# Sketch to SwiftUI
+
+> Create a StylableSwiftUI stylist implementation from a Sketch file. Sketch -> SwiftUI
+
+## Installation
+
+- Clone this repository
+- Run `npm i`
```

Comment:
Hmm, I/we should push to npm then we can just say run `npm i -g sketch-to-swiftUI` and then `sketch-to-swiftUI <file>`